### PR TITLE
Bugfix for PV is deleted before PVC

### PR DESCRIPTION
**Pull request by qinpingli**
Patch for v3 branch.

@duanwei33 @chao007 Please help review.

### CODE REVIEW

File: features/storage/nfs.feature [hunk 1]
```diff
@@ -135,3 +139,5 @@ Feature: NFS Persistent Volume
     And the output should contain:
       | Permission denied |
 
+   Given I ensure "nfspd-<%= project.name %>" pod is deleted
+   And I ensure "nfsc-<%= project.name %>" pvc is deleted
```
Comment (akostadinov) [inline features/storage/nfs.feature:143]: I don't mind these. But if test fails, we never reach these steps. Shouldn't we register deletion `after scenario` earlier as clean-up steps?

Comment (akostadinov) [reply features/storage/nfs.feature:143]: @jhou1 ?

**Comment by akostadinov** [comment]
How do things fail before this patch? Shall we fix standard clean-up logic instead?

**Comment by akostadinov** [comment]
/lgtm

to unblock but I think a better fix is preferred.

File: features/storage/nfs.feature (hunk 1 above)
Comment (liangxia) [reply features/storage/nfs.feature:143]: Added that in https://github.com/openshift/verification-tests/pull/1562

**Pull request merged**